File: PHP/UberX.php
<?php
    class UberX extends Car{
        private $brand;
        private $model;

        public function __construct($license, $driver, $brand, $model)
        {
            parent::__construct($license, $driver);
            $this->license = $license;
            $this->driver = $driver;
        }
    }
?>

File: Java/UberX.java
class UberX extends Car{
    String brand;
    String model;

    public UberX(String license, Account driver, String brand, String model){
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }

    
    @Override
    public void mostrarDatos() {
        super.mostrarDatos();
        System.out.println("Marca: " + brand + " | Modelo: " + model);
    }
}
